roles/lib_openshift/src/class/oadm_policy_group.py

```diff
@@ -0,0 +1,180 @@
+# pylint: skip-file
+
+class OadmPolicyGroupConfig(OpenShiftCLIConfig):
```

Comment:
To follow the other ported modules, should rename to `OCAdm` or remove `Oadm` from the signatures.